src/SFA.DAS.EmployerAccounts.Jobs/StructureMapJobActivator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Azure.WebJobs.Host;
+using StructureMap;
+
+namespace SFA.DAS.EmployerAccounts.Jobs
+{
+    public partial class Program
```

Comment:
Is this needed?

Reply:
No, I don't think so, but it appeared after a merge from MA. Personally I think partial classes should be banned